Sources/AppcuesKit/Presentation/Traits/Appcues/AppcuesTooltipTrait.swift

```diff
@@ -27,10 +27,10 @@ internal class AppcuesTooltipTrait: StepDecoratingTrait, WrapperCreatingTrait, P
     let pointerSize: CGSize
 
     required init?(configuration: ExperiencePluginConfiguration, level: ExperienceTraitLevel) {
-        guard let config = configuration.decode(Config.self) else { return nil }
-        self.hidePointer = config.hidePointer ?? false
-        self.pointerSize = CGSize(width: config.pointerBase ?? 16, height: config.pointerLength ?? 8)
-        self.tooltipStyle = config.style
+        let config = configuration.decode(Config.self)
+        self.hidePointer = config?.hidePointer ?? false
```

Comment:
should we note this default `false` [here](https://github.com/appcues/appcues-mobile-experience-spec/blob/main/trait-schemas/appcues-tooltip-schema.json#L21)? It does seem pretty obvious in this case that it would not be hidden by default for a standard tooltip.

Reply:
Yeah, explicitness is always good. We have an unwritten rule that our config booleans are named in such a way that they're option and default to false, but good to be clear about it!

Reply:
https://github.com/appcues/appcues-mobile-experience-spec/pull/140